### PR TITLE
2021 Food Revamp

### PR DESCRIPTION
**Pull request by GhostActual**
Polishes, adds, and changes some sprites for food and snacks alike that were a bit outdated and needed updating. Some organizational changes were also added to the code and sprite files for better reference and a `top_bun` and `bottom_bun` was added to the `food.dmi` file for any future burger additions.

Below are some examples of the sprites that were changed/added.

Onion Rings:

Cuban Carp:

Old Pizza:

Cooked Bacon:

New BBQ Chip design and Salt & Vinegar Chips:
Sprite-0001

New Caramel Apple and Candy Apple:
Sprite-0002

New Condiment Sprites:
125782699-a40acbe3-b63c-47f4-b3bb-9ca11bd720c7

New Baked Bean can:

New Burritos:
showcase

New Tacos:
taco_showcase

New Honeycomb:
showcase

New Spacer Cake Sprites (nothing major, just adds cake_out_of_package sprite):
Sprite-0001

New Hot Dogs:
hotdog_showcase

New Packaged Food (only spawns in SnL vendor):
Showcase

Misc Stuff:
-New Vinegar reagent added with condiment bottles
-New Mustard reagent added with condiment bottles
-Makes all plate/bowl sprites symmetrical for consistency
-Adds new `small_bowl` trash sprite for Wing Fang Chu (can be used for future soups in small bowls)
-Adds new `stew` sprite that is more symmetrical and straight
-Changes worm can sprites slightly to round them out and make them look similar to other cans

### CODE REVIEW

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -94,6 +98,10 @@
 	name = "snack bowl"
 	icon_state	= "snack_bowl"
 
+/obj/item/trash/small_bowl
+	name = "small bowl"
+	icon_state	= "small_bowl"
+
```
Comment (Cerebulon) [inline code/game/objects/items/trash.dm:104]: Love the sprites, but this (and the bowl above it which was an oversight) should probably use
drop_sound = 'sound/items/drop/food.ogg'
pickup_sound = 'sound/items/pickup/food.ogg'

File: code/modules/economy/vending_machines.dm [hunk 2]
```diff
@@ -898,7 +900,8 @@
 					/obj/item/weapon/reagent_containers/food/snacks/sun_snax = 8,
 					/obj/item/weapon/reagent_containers/food/snacks/canned/appleberry = 6,
 					/obj/item/weapon/storage/box/gum = 8,
-					/obj/item/weapon/storage/box/admints = 8
+					/obj/item/weapon/storage/box/admints = 8,
+					/obj/item/weapon/reagent_containers/food/snacks/canned/ntbeans = 6
```
Comment (Cerebulon) [inline code/modules/economy/vending_machines.dm:904]: A tin of beans is definitely not something you get from any 'normal' vending machine?

**Comment by Cerebulon** [comment]
wha

**Comment by GhostActual** [comment]
> wha

Sorry, was too tired to give a reason, I'm working directly with the `Vorestation` database right now and I've currently overhauled more food there than I have here. From what I read there - there are more sprites on `Polaris` that I have to update and add as practically every sprite in `food_syn.dmi` needs changed as well. There are still more sprites, and by the looks of it I'll need to redo the entire PR.

**Comment by Cerebulon** [comment]
Any plans to reopen this with the changes you finished downstream, or should I port it myself?

**Comment by GhostActual** [comment]
> Any plans to reopen this with the changes you finished downstream, or should I port it myself?

If you feel that you're up for the challenge, be my guest, I still have other PRs that are pending review and I don't want to open anymore until they're cleared

**Comment by Cerebulon** [comment]
It looks like all the open ones this end are DNM because there's .dmi conflicts you'll need to resolve (Though besides that ater hasn't had time to review much this week) but if you're busy then I can just grab the merged virgo version of this and port it over with tweaks.

**Pull request closed without merging**